### DISCUSSION TITLE
gcc-11 stops with message 'Killed : 9'

### DISCUSSION THREAD

**Discussion by sriharisangaraju** [Homebrew/discussions#2925] [Everyday usage] [answered]
I am new to Mac environment, and using Macbook Pro with M1 pro chip. 

I tried installing gcc and dependencies by 'build-from-source' option. I have not received any error messages during compilation of gcc-11 from Homebrew. The executables gcc-11, gfortran-11 are generated, also 'brew doctor' doesnt show any warnings.

When I run 'gcc-11 --version', bash shell shows "Killed : 9". 
where as zsh shell shows "zsh: killed     gcc-11 --version".

Please let me know, if you like to see any log files.

P.S. The gcc-11 installed using Homebrew bottle is running, but I was thinking, compiling 'gcc' on my own computer has some performance benefits.

Thanks,
Srihari

**Comment by carlocab** [comment, marked as answer]
You should install all formulae from the bottles. There are no performance benefits to be gained from compiling software for yourself on M1 Macs. There may be on Intel Macs -- but not without patching `brew`.
